lib/gui/app.js

```diff
@@ -28,9 +28,12 @@ var angular = require('angular');
 
 const electron = require('electron');
 const Bluebird = require('bluebird');
+const semver = require('semver');
+const _ = require('lodash');
 const EXIT_CODES = require('../shared/exit-codes');
 const messages = require('../shared/messages');
-const packageJSON = require('../../package.json');
+const s3Packages = require('../shared/s3-packages');
+const information = require('../shared/information');
```

Comment:
True. I can't come up with a good name though... `buildInformation` and `versionInformation` still sound weird. Maybe we should remove `information` from it?